tfx/dsl/component/experimental/testdata/kubeflow_container_component_test.yaml

```diff
@@ -0,0 +1,26 @@
+name: Test_KubeFlow_Container_Component
+description: Component spec for testing kubeflow_container_component
+metadata:
+  annotations: {annotation1: annotation1, annotation2: annotation2}
+  labels: {labels1: labels1}
+inputs:
+- {name: input1, type: Directory, description: description1, default: default1, optional: false}
+- {name: input2, type: String, description: description2, default: default2, optional: true}
+outputs:
+- {name: output1, type: File, description: description3}
+implementation:
+  container:
+    image: image1
+    command:
+    - command1
+    - command2
+    - command3
+    - inputPath: Directory
+    - inputValue: Subpath
+    - outputPath: File
+    args: [
+      --arg1, {inputValue: input3},
+      --arg2, {inputPath: input4},
+      --arg3, {outputPath: output2},
```

Comment:
Correct me if I'm wrong here but I think the intention is that these specify the inputs / outputs already declared above?

Reply:
Yes, I've made the corresponding changes to make them consistent